### PR TITLE
Update web again

### PR DESCRIPTION
**Pull request by ljacobson64**

### CODE REVIEW

File: doc/usersguide/index.rst [hunk 1]
```diff
@@ -3,43 +3,40 @@ User's guide
...
-
-**N** not supported, **A** automatic production at run time, **M** manually performed
-
-So, if you are interested in running your geometry in multiple physics 
-packages with minimal effort, the UW2 workflow is your best option
-as it does the most work for you, however if you are interested
-in only a specific physics engine one of the basic workflows may
-interest you more. We still would recommend the UW2 workflow as it 
-offers robust and tested methods for material creation.
+This document is intended for users of the DAGMC toolkit who want to better
```
Comment (makeclean) [inline doc/usersguide/index.rst:6]: Can this block go back to how it was, the abbreviated form of the table was preferred.

File: doc/usersguide/postprocessing.rst [hunk 2]
```diff
@@ -37,33 +38,33 @@ of this is shown below, curves are shown on the left and facets on the right.
...
-The curve vtk is useful if you wish to examine the final model in a convenient 
-and graphically responsive method, however one cannot perform slice operations
-on the curve file. When plotting alongside mesh files, the stl file should be 
-used as this can be conviently sliced. If the stl file is still prohibitively large,
-it is sometimes useful to generate a facet file with higher tolerances purely
-for vizualization purposes.
+The .vtk file containing the curves is useful if you wish to examine the final
+model in a convenient and graphically responsive method; however, one cannot
+perform slice operations on the curve file. When plotting alongside mesh files,
+the .stl file should be used as it can be sliced conviently. If the .stl file is
```
Comment (makeclean) [inline doc/usersguide/postprocessing.rst:44]: *conveniently*

**Review by makeclean — CHANGES_REQUESTED** [review]
When you've fixed those, ill build and take a look at the rendered version

File: doc/usersguide/index.rst (hunk 1 above)
Comment (ljacobson64) [reply doc/usersguide/index.rst:6]: Fixed

File: doc/usersguide/postprocessing.rst (hunk 2 above)
Comment (ljacobson64) [reply doc/usersguide/postprocessing.rst:44]: Changed this to "in a convenient and graphically responsive way"

**Review by makeclean — APPROVED** [review]

**Comment by makeclean** [comment]
Thanks @ljacobson64

**Pull request merged**